src/main/java/com/epam/reportportal/auth/integration/github/GitHubUserReplicator.java

```diff
@@ -156,7 +155,7 @@ private void fillDefaultUserData(User user) {
 		user.setUserType(UserType.GITHUB);
 		user.setRole(UserRole.USER);
 		user.setExpired(false);
-		user.setDefaultProject(generatePersonalProject(user));
+		generatePersonalProject(user).getUsers().stream().findFirst().ifPresent(it -> user.getProjects().add(it));
```

Comment:
probably it's better to split logic of project adding to the user
cause from the first watch it doesn't really clear to understand what is going on here.